site/developer/developer_guide.md

```diff
@@ -0,0 +1,238 @@
+# Developer Guide
+
+## Table of Contents
+- [Project Structure](#project-structure)
+- [Functional Areas](#functional-areas)
+- [Alias Definitions](#alias-definitions)
+- [Building WebLogic Deploy Tool](#building-weblogic-deploy-tool)
+
+## Project Structure
+
+This project is structured using the Standard Directory Layout for Maven projects, with two child modules, `core` and `installer`. In addition, there is a `samples` directory with example configurations, and the `site` directory containing project documentation.
+
+The `core` module contains the main source code for the project. This includes Jython modules and Java classes, as well as typedef files, alias definitions, and the message bundle. There are unit tests related to this module.
+
+Alias definitions are discussed in more detail [here](#alias-definitions). 
+
+The `installer` module builds the final installer zip file. It includes the assembly definitions, start scripts for each tool for Linux and Windows platforms, and configurations for variable injection and logging. 
+
+A single installer zip file is built under the `WLSDEPLOY_HOME/installer/target` directory.
+
+There are detailed instructions for building the project [here](#building-weblogic-deploy-tool)
```

Comment:
[here](#building-weblogic-deploy-tool) -> [here](#building-weblogic-deploy-tool). (period, end of sentence)

Reply:
check